snorkel/map/core.py

```diff
@@ -52,7 +128,10 @@ def set_mode(self, mode: MapperMode) -> None:
         """
         self.mode = mode
```

Comment:
Ditto here re: comment above

Reply:
Ooooh you shouldn't have highlighted this one...now I want to leave these methods for proper type checking.

Reply:
Scratch that, mypy will yell if we type the attribute.